### PR TITLE
BIP70 Payment Protocol Multiple Outputs

### PR DESCRIPTION
**Pull request by vinarmani**
The spec for BIP70 allows for multiple outputs in the payment request. This functionality was, for some reason, just ignored in the bitcore libraries. Both CoPay wallet and Bitcoin.com wallet have never supported it. This adds full support.

This update is backwards compatible (for only one output) with the bitcore-wallet-client fork maintained by Bitcoin.com. However, there are some minor changes to that module required to allow multiple outputs. I have made those updates and made a pull request at https://github.com/Bitcoin-com/bitcore-wallet-client/pull/1 . In the meantime, I have pointed package-template.json at my fork of bitcore-wallet-client.

### CODE REVIEW

**Comment by vinarmani** [comment]
Can test single output with:
bitcoincash:?r=https://pay.cointext.io/i/ZKHOR (expires 3/14)

Can test multiple outputs with:
bitcoincash:?r=https://pay.cointext.io/i/UQCKE (expires 3/29)

**Comment by pysmitty38** [comment]
Not to get off the subject but I'm looking deeper into a marketing site and an option in getting earnings may be different currencies. Wallet I have and thought of using. Take a look.. 

https://twitter.com/pysmith78/status/1108580573126250496?s=20

**Comment by vinarmani** [comment]
If the below two pull requests:

https://github.com/Bitcoin-com/bitcore-payment-protocol/pull/1
https://github.com/Bitcoin-com/bitcore-wallet-client/pull/1

Are done in that order, then the dependencies (in the wallet and wallet-client) can remain pointing at the modules in the Bitcoin.com github repos.

File: app-template/package-template.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     "bezier-easing": "^2.0.3",
     "bhttp": "1.2.1",
     "bitauth": "^0.2.1",
-    "bitcore-wallet-client": "https://github.com/Bitcoin-com/bitcore-wallet-client.git",
+    "bitcore-wallet-client": "https://github.com/vinarmani/bitcore-wallet-client.git",
```
Comment (jbdtky) [inline app-template/package-template.json]: Please, change to Bitcoin-com and I will merge to our task branch. Then we will manage this part in our side until production.

Comment (vinarmani) [reply app-template/package-template.json]: Done

**Pull request merged**